### PR TITLE
update itools config for cse in relation to farao core upgrade, and f…

### PR DESCRIPTION
**Pull request by schwitzguebel**
…ix bug in import_ec ntc2 fr databridge

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by TheoPascoli — APPROVED** [review]

**Review by kahyami — CHANGES_REQUESTED** [review]

File: k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml [hunk 1]
```diff
@@ -12,49 +12,61 @@ data:
...
+    rao-not-optimized-cnecs:
+      do-not-optimize-curative-cnecs-for-tsos-without-cras: false
+
+    rao-mnec-parameters:
+      acceptable-margin-decrease: -99999
+      violation-cost : 100.0
+      constraint-adjustment-coefficient : 5.0
+      
+    rao-ra-usage-limits-per-contingency:
+      max-curative-ra-per-tso: [ "{IT}:6", "{FR}:5", "{CH}:1", "{AT}:3", "{SI}:3" ]
```
Comment (WalAmeni) [inline k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml:72]: Not used for the moment for the export

File: k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml [hunk 2]
```diff
@@ -12,49 +12,61 @@ data:
...
-      curative-rao-optimize-operators-not-sharing-cras : true
\ No newline at end of file
+
+    rao-load-flow-and-sensitivity-computation:
+      load-flow-provider: OpenLoadFlow
+      sensitivity-provider: OpenLoadFlow
+
+    rao-topological-actions-optimization:
+      max-search-tree-depth: 3
+      predefined-combinations: [ "{PRA_2N_VALPELLINE}+{PRA_2N_AVISE}" ]
```
Comment (WalAmeni) [inline k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml:54]: Not used for the export

**Review by WalAmeni — CHANGES_REQUESTED** [review]

File: k8s/overlays/azure/test/cse-gridcapa-export-runner-itools-azure-test-configmap.yaml [hunk 3]
```diff
@@ -18,45 +18,54 @@ data:
...
-      newtonRaphsonConvEpsPerEq : 1.0E-2
-      minRealisticVoltage: 0.4
-      maxRealisticVoltage: 1.6
-
-    rao-parameters:
-      objective-function: MAX_MIN_MARGIN_IN_AMPERE
-      max-number-of-iterations: 5
-      optimization-solver: XPRESS
+      newtonRaphsonConvEpsPerEq: 1.0E-2
+
```
[the remark this answers is not in the document]
Comment (schwitzguebel) [reply k8s/overlays/azure/test/cse-gridcapa-export-runner-itools-azure-test-configmap.yaml:31]: sorry

File: k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml (hunk 2 above)
Comment (schwitzguebel) [reply k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml]: ok

File: docker-compose/cse-common/cse-gridcapa-itools-config.yml [hunk 4]
```diff
@@ -1,44 +1,54 @@
...
-  curative-rao-optimize-operators-not-sharing-cras : true
\ No newline at end of file
+
+rao-load-flow-and-sensitivity-computation:
+  load-flow-provider: OpenLoadFlow
+  sensitivity-provider: OpenLoadFlow
+
+rao-topological-actions-optimization:
+  max-search-tree-depth: 3
+  predefined-combinations: [ "{PRA_2N_VALPELLINE}+{PRA_2N_AVISE}" ]
```
[the remark this answers is not in the document]
Comment (schwitzguebel) [reply docker-compose/cse-common/cse-gridcapa-itools-config.yml:41]: ok

File: k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml (hunk 1 above)
Comment (schwitzguebel) [reply k8s/overlays/coreso/test/cse-gridcapa-runner-itools-coreso-configmap.yaml]: ok

**Review by kahyami — APPROVED** [review]

**Review by WalAmeni — APPROVED** [review]

**Pull request merged**